cmd/soroban-rpc/internal/methods/get_transactions.go

```diff
@@ -58,6 +58,8 @@ func (req GetTransactionsRequest) isValid(maxLimit uint, ledgerRange ledgerbucke
 type TransactionInfo struct {
 	// Status is one of: TransactionSuccess, TransactionFailed.
 	Status string `json:"status"`
+	// TransactionHash is the hex encoded hash of the transaction.
+	TransactionHash string `json:"txHash"`
```

Comment:
Thoughts on just encoding it to `hash`? The `tx` prefix is implied by it being, well, a transaction structure.

Reply:
agree with just `hash` but `EventInfo`  encodes it as `txHash`  so let's just stay consistent.

Reply:
@Shaptic @psheth9 I was also thinking of hash but saw the txHash in events and wanted to be consistent with it as you mentioned

Reply:
I will keep this as `txHash`